sdks/java/container/build.gradle

```diff
@@ -60,8 +60,11 @@ golang {
 }
 
 docker {
-  // TODO(herohde): make the name easier to generate for releases.
-  name System.properties['user.name'] + '-docker-apache.bintray.io/beam/java:latest'
+  def repositoryRoot = System.properties["docker-repository-root"]
```

Comment:
+1 on using project properties.

It allows people to setup their own gradle.properties file.

Reply:
Done. Although it looks like `gradle.properties` is checked into git. How do we add personal properties?

Reply:
Set GRADLE_USER_HOME.

From https://docs.gradle.org/current/userguide/build_environment.html#sec:gradle_configuration_properties

Setting up a consistent environment for your build is as simple as placing these settings into a gradle.properties file. The configuration is applied in following order (if an option is configured in multiple locations the last one wins):
* gradle.properties in project root directory.
* gradle.properties in GRADLE_USER_HOME directory.
* system properties, e.g. when -Dgradle.user.home is set on the command line.

Reply:
Thanks for the reference!